lib/v2.recommended.yaml

```diff
@@ -94,42 +106,57 @@ analyzer:
     prefer_asserts_in_initializer_lists: warning
     prefer_collection_literals: warning
     prefer_conditional_assignment: warning
+    prefer_const_constructors_in_immutables: warning
     prefer_const_declarations: warning
-    provide_deprecation_message: warning
+    prefer_const_literals_to_create_immutables: warning
+    prefer_constructors_over_static_methods: warning
+    prefer_typing_uninitialized_variables: warning
     prefer_final_fields: warning
     prefer_final_in_for_each: warning
     prefer_final_locals: warning
     prefer_function_declarations_over_variables: warning
     prefer_if_elements_to_conditional_expressions: warning
     prefer_initializing_formals: warning
     prefer_inlined_adds: warning
+    prefer_int_literals: warning
     prefer_interpolation_to_compose_strings: warning
     prefer_iterable_whereType: warning
     prefer_mixin: warning
     prefer_null_aware_operators: warning
     prefer_single_quotes: warning
     prefer_spread_collections: warning
     prefer_void_to_null: warning
+    provide_deprecation_message: warning
     sort_pub_dependencies: warning
```

Comment:
This is also ignored, primarily I believe due to the subpackage structure that exists.